### DISCUSSION TITLE
BT-13721 indicator missing

### DISCUSSION THREAD

**Discussion by mgakt** [OP-TED/eForms-SDK#718] [Q&A] [answered]
Dear All,
The BT-13721 indicator is missing from the SDK, but in 1780/2019, it is mandatory filed in numerous notice types. This incoherence must be handled on a technical or legal level.

**Comment by YvesJo** [comment, marked as answer]
The Regulation Annex, in its excel format, is a "normalized" representation that requires "XYZ Lot Identifier" to keep the association of objects with the lots. The XML representation is a "de-normalized" one and for each Lot, the Strategic Procurement associated fields may be found inside the Lot itself. You may find a list of "useless" BTs at https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html#pointlessDueToDesignSection
